symengine/lib/symengine_wrapper.pyx

```diff
@@ -427,8 +427,10 @@ def sympy2symengine(a, raise_error=False):
         return piecewise(*(a.args))
     elif isinstance(a, sympy.Interval):
         return interval(*(a.args))
-    elif isinstance(a, sympy.EmptySet):
+    elif isinstance(a, type(sympy.S.EmptySet)):
```

Comment:
@asmeurer, this would work in previous sympy versions and after https://github.com/sympy/sympy/issues/16337 is resolved right?